v1.15/alpine/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# AUTOMATICALLY GENERATED
+# DO NOT EDIT THIS FILE DIRECTLY, USE /Dockerfile.template.erb
+
+FROM alpine:3.16
+LABEL maintainer "Fluentd developers <fluentd@googlegroups.com>"
+LABEL Description="Fluentd docker image" Vendor="Fluent Organization" Version="1.15.0"
+
+# Do not split this into multiple RUN!
+# Docker creates a layer for every RUN-Statement
+# therefore an 'apk delete' has no effect
+RUN apk update \
+ && apk add --no-cache \
+        ca-certificates \
+        ruby ruby-irb ruby-etc ruby-webrick \
+        tini \
+ && apk add --no-cache --virtual .build-deps \
+        build-base linux-headers \
+        ruby-dev gnupg \
+ && echo 'gem: --no-document' >> /etc/gemrc \
+ && gem install oj -v 3.10.18 \
+ && gem install json -v 2.4.1 \
+ && gem install async-http -v 0.54.0 \
+ && gem install ext_monitor -v 0.1.2 \
```

Comment:
We must remove `ext_monitor` installation step if we move to use Ruby 3.0 or later.
ref: https://rubygems.org/gems/ext_monitor

Reply:
I updated Ruby to 3.1 and removed ext_monitor. Thanks